.github/workflows/release.yaml

```diff
@@ -6,6 +6,7 @@ on:
       - main
     paths:
       - 'CHANGELOG/*.md'
+  workflow_dispatch:
```

Comment:
Is this something we expect to keep, or is this just to get over the hump of this one failed job?

Reply:
It's specifically so we don't have to create a no-op PR to trigger the job this time, but seems like it could useful to retain as an escape hatch.

Reply:
Will the changed-files action pick up the CHANGELOG file then for a manually-triggered run?

Reply:
Erm...maybe? I'm not sure actually. 🤦🏻 
I guess in that case we can fall back to creating another PR that makes a trivial change to the CHANGELOG (and revert `workflow_dispatch`).

Reply:
Good catch!  I just tested on my fork and you're right it doesn't detect files.  I will remove this.